toolbox

```diff
@@ -173,35 +171,55 @@ list() {
     exit $?
 }
 
+stop() {
+    # We can't stop non-existing and non-running toolboxes
+    if  ! container_exists ; then
+        echo "$0: ERROR: Cannot stop non-existing container '$TOOLBOX_NAME'"
+        exit 2
+    fi
+    if [[ "$(container_state)" != "running" ]]; then
+        echo "$0: ERROR: Cannot stop non-running container '$TOOLBOX_NAME'"
+        exit 2
+    fi
+
+    # We don't stop toolboxes with active sessions
+    if [ $(container_active) -ne 0 ]; then
+        echo "$0: ERROR: The toolbox '$TOOLBOX_NAME' has active sessions. Not stopping"
+        exit 1
+    fi
+
+    ${SUDO} $CLI stop "$TOOLBOX_NAME" &> /dev/null
+}
+
 container_create() {
+    if [ -z "$SANDBOX" ]; then
+        # this is the default behavior, unless --sandbox is specified
+        CREATE_NO_SANDBOX="--privileged --volume /dev:/dev:rslave --volume /sys:/sys:rslave --volume /:/media/root:rslave"
+        CREATE_NO_SANDBOX="$CREATE_NO_SANDBOX --privileged --security-opt label=disable --pid host --ipc host"
```

Comment:
Hi @dfaggioli 
in these lines `--privileged` is specified twice

Reply:
Oh, wow, it is! Thanks for checking and reporting... Should be fixed now.